src/lib/vaccineCodes.js

```diff
@@ -0,0 +1,14 @@
+var _vaccineCodeMapping = undefined;
+
+export async function getVaccineCodeMapping() {
+  if (_vaccineCodeMapping) return _vaccineCodeMapping;
+  try {
+    const response = await fetch(
+      "https://raw.githubusercontent.com/hellodocket/vaccine-code-mappings/main/vaccine-code-mapping.json"
```

Comment:
I would have preferred to use the repo's release assets, but Github protects that with CORS.